Source/WebKit/Shared/EditorState.h

```diff
@@ -48,23 +48,23 @@ class TextStream;
 
 namespace WebKit {
 
-enum TypingAttributes {
+enum class TypingAttribute : uint16_t {
     AttributeNone = 0,
```

Comment:
these enum cases should probably have their Attribute prefix removed to reduce redundancy